Article.md

```diff
@@ -0,0 +1,178 @@
+acl_grant_role("Resetter", "alice.near");
+```
+
+The `AccessControllable` trait provides many more methods to administer ACL permissions. After following the steps above, all of them are automatically implemented for a contract using the `AccessControllable` plugin.
+
+# Done!
+
+The steps above are sufficient to add complex and configurable ACL permissions to a contract using `near-plugins`. At this point, `alice.near` is the only account which has been granted the `Resetter` role. This means that only `alice.near` may successfully call the contract’s `reset()` method.
+
+The repo contains an [integration test](https://github.com/mooori/counter-acl-example/blob/4a526533c528f09ca436b932e84c37594cbda296/tests/workflow.rs) which verifies that `AccessControllable` was setup up correctly for our `Counter` contract. Take a look at it to learn more about interacting with an `AccessControllable` contract. To run the test on-chain in a local sandbox, it suffices to clone the repo and execute the following command. This is made possible by [near-workspaces-rs](https://github.com/near/near-workspaces-rs).
+
+```
+cargo test
+```
+
+## Teaser: How it works internally
+
+Using `AccessControllable` extends the contracts state to store the permissions that have been granted. Moreover, the `AccessControllable` trait is implemented for the contract to enable administering permissions. When `#[access_control_any(roles(...))]` is attached to a method, `near-plugin` injects code which checks if the caller was granted any of the required roles. If not, a panic is generated which aborts the function call.
+
+To learn about all the details, you can dive into the [implementation](https://github.com/aurora-is-near/near-plugins/blob/master/near-plugins-derive/src/access_controllable.rs) of the `AccessControllable` macro.
+## A note on testing
+
+The functionality provided by `near-plugins` is critical for security and we strive to test it exhaustively. In tests we compile demo contracts for all plugins and deploy them on-chain in a local sandbox. Then we verify that using a particular plugin adds exactly the expected functionality to the contract. These tests and demo contracts can be found [here](https://github.com/aurora-is-near/near-plugins/tree/master/near-plugins-derive/tests).
+## Ready for production, though?
+
+As mentioned earlier, `near-plugins` comes with the caveat of not yet being published to crates.io. Nevertheless, we are confident that `near-plugins` can already be used in production. In fact, smart contracts related to the [Rainbow Bridge](https://rainbowbridge.app/transfer) already use `near-plugins`. Moreover, both [Hacken](https://www.datocms-assets.com/50156/1680101850-hacken-near-plugins-final-report-updated-march2023.pdf) and [AuditOne](https://www.datocms-assets.com/50156/1680590522-auditone-near-plugins-final-report-updated-march2023.pdf) audited `near-plugins`, awarding it high scores.
```

Comment:
> we are confident that `near-plugins` can already be used in production

Hmm... I am a bit vary of the legal implications of such a statement.  I am not a lawyer, but still ca we make this less strong maybe?

Reply:
How about something like: As mentioned earlier, `near-plugins` comes with the caveat of not yet being published to crates.io.  It is currently used in some contracts on the mainnet e.g. ....

Reply:
Adapted it according to your proposal in 40e153b.